gpMgmt/test/behave/mgmt_utils/gpexpand.feature

```diff
@@ -804,3 +804,20 @@ Feature: expand the cluster by adding more segments
         And table "test_good_2" should be marked as expanded
         And table "test_already_expanded" should be marked as expanded
         And table "test_broken" should not be marked as expanded
+
+    @gpexpand_avoid_tarfile_overwrite
+    Scenario: Avoid overwriting the tar file on coordinator
+        Given the database is not running
+        # need to remove this log because otherwise SCAN_LOG may pick up a previous error/warning in the log
```

Comment:
And then these logs won't be needed anywhere if we want to study something in CI?

Reply:
Every test in this file uses these lines. I just copied them. Looks like a separate problem.